tests/dragonfly/pymemcached_test.py

```diff
@@ -33,3 +35,17 @@ def test_mixed_reply(memcached_connection):
     memcached_connection.add(b"key", b"final", noreply=True)
 
     assert memcached_connection.get(b"key") == b"data"
+
+
+@dfly_args({"memcached_port": 11211})
+def test_length_in_set_command(df_server: DflyInstance):
```

Comment:
looks good but since we had the iteration with faulty implementation that rejected blobs with \r\n in them, lets also add the case that covers this too.